### PR TITLE
Setup New Proposal email (read before merge)

### PR DESCRIPTION
**Pull request by rememberlenny**
# Note

This comes after https://github.com/18F/C2/pull/1011 and https://github.com/18F/C2/pull/1012

# Design note

Authorization list is wider than the design. The design breaks w/ outlook, if the name listed is a long email address. The area is now wider to cater to a email address.

o
chromegmailnew-vertical-allowed-1366 2
ol2013-vertical-allowed-1366 2

### CODE REVIEW

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -23,4 +23,5 @@ def display_profile_warning?
   def display_search_ui?
     current_user && current_user.client_model && !client_disabled?
   end
+
```
Comment (jessieay) [inline app/helpers/application_helper.rb]: looks like you added newlines after blocks in a few files - I generally like newlines before/after multi-line blocks *except* when the end the block is directly nested inside another block (eg: two `end`s next to each other) what do you think?

**Comment by jessieay** [comment]
I'm having a little trouble figuring out if some of these changes are included because they are part of commits that this PR depends on or if they are part of the PR itself. GitHub should auto-update to only show the commits for this PR since the two it depends on are merged, but I've noticed recently that it doesn't always work that way (Perhaps because we are merging into a non-master branch?)

Which Trello card does this PR represent? If you can let me know then I think it will be easier to figure it out. Also re-rebasing against `EMAIL-REWRITE-MASTER` would be cool

Thanks! this all looks awesome

**Comment by rememberlenny** [comment]
@jessieay Email is from this ticket: https://trello.com/c/lf0XzPWo/120-1-email-conversion-requester-new-request-confirmation

**Comment by jessieay** [comment]
thanks for the update, @rememberlenny -- mergin!

**Pull request merged**